go/api_statement.go

```diff
@@ -167,7 +164,8 @@ const (
 //
 // It polls the server until the query is finished, with fix delay of 1 second.
 func (f *ResultSetFetcher) FetchResultSet(ctx context.Context) (*StatementResponse, error) {
-	ticker := time.NewTicker(defaultFetchInterval)
+	tick := 10 * time.Millisecond
```

Comment:
Start with 100ms?

Reply:
OK. Anyway we'd implement synchornized wait at the server side for small query later >_<